src/main/java/io/lettuce/core/api/async/RedisSortedSetAsyncCommands.java

```diff
@@ -1258,4 +1258,80 @@
      * @return Long integer-reply the number of elements in the resulting sorted set at {@code destination}.
      */
     RedisFuture<Long> zunionstore(K destination, ZStoreArgs storeArgs, K... keys);
+
+    /**
+     * Add multiple sorted sets and returns the resulting sorted set.
+     *
+     * @param numkey the numeric key.
+     * @param keys the keys.
+     * @return List&lt;V&gt; array-reply list of elements.
+     */
+    RedisFuture<List<V>> zunion(long numkey, K... keys);
```

Comment:
New methods should have a `@since` tag. Those commands will be shipped with Lettuce 6.1, so please add `@since 6.1` to every public API method.

Reply:
fixed